test/lib/reports/employee_leave_report_test.rb

```diff
@@ -16,10 +17,24 @@ class EmployeeLeaveReportTest < ActiveSupport::TestCase
   end
 
   test 'comparision report of employees' do
+    report_data
     sign_offs_report = Reports::EmployeeLeaveReport.new.comparision_report(@year)
     sign_offs = @user.sign_offs.includes(:sign_off_type).where("extract(year from date_from) = ?", @year)
-    assert_equal sign_offs.count, sign_offs_report['john'].values.inject(:+)
+    assert_equal sign_offs.count, sign_offs_report['johnna'].values.inject(:+)
     sign_off_type_expected = sign_offs.pluck(:'sign_off_types.sign_off_type_name').inject(Hash.new(0)) { |total, e| total[e] += 1 ;total}
-    assert_equal sign_off_type_expected, sign_offs_report['john']
+    assert_equal sign_off_type_expected, sign_offs_report['johnna']
+  end
+
+  private
+
+  def report_data
```

Comment:
Is it already defined somewhere? If the definition is same then move it to test_helper and use anywhere.